gnomad_qc/analyses/compute_pext.py

```diff
@@ -224,6 +225,20 @@ def main(args):
     ht = pext_res.context_ht.ht().naive_coalesce(5000)
     tx_mt = pext_res.tx_mt.mt()
 
+    # Any tissue below 50 samples will NOT get shown on website in overall pext or as
+    # individual tissue.
+    # Exclude testes and cell lines from overall pext but keep in display as
+    # individual tissues (agreed to take a look at the per tissue pext values to make
+    # sure this makes sense).
```

Comment:
```suggestion
    # Exclude testes and cell lines from overall pext but keep in display as
    # individual tissues (checked per tissue pext values to make
    # sure this made sense).
```
changing tense since this was done, right?

Reply:
Sorry, this comment was supposed to get deleted. I copied what I wrote in Slack so that I knew what I was supposed to change things to. I will fix